common/models/transducer/topology.py

```diff
@@ -0,0 +1,48 @@
+from .loss import (rnnt_loss, rnnt_loss_out_type,
+                   rnnt_tf_loss, rna_tf_loss, rna_loss_out_type)
+from .alignment import (rnnt_alignment, rnnt_alignment_out_type,
+                        rna_alignment, rna_alignment_out_type)
+
+
+class Topology:
+  """
+  Class to hold informations about different topologies such as loss-, alignment-funcion and their Data_out_types.
+  loss and alignment functions are to be used in eval like layers that return a source function.
+  The loss and alignment function both are to be used in EvalLayers taking from layers that output the followings:
+        output_log_prob: (B, T, U+1, V) log-probabilities
+        target: (B, U) -> [V] target labels
+        base:encoder: (B, T, Feat) -> [V] encoder output
+  """
+  def __init__(self, name: str,
+               loss,
+               loss_out_type,
+               alignment,
+               alignment_out_type):
+    """ Label Topology such as rnnt, rna, ctc.
+    Args:
+        name (str)
```

Comment:
You are still ignoring what I wrote.
```suggestion
```

Reply:
added the information about the shapes under the class name. So that i dont repeat the same things

Reply:
See my suggestion above:
> Suggested change 
```
-         name (str)
```
That means to remove this line.